napari_aicssegmentation/controller/workflow_steps_controller.py

```diff
@@ -8,11 +7,12 @@
 
 @debug_class
 class WorkflowStepsController(Controller, IWorkflowStepsController):
-    def __init__(self, application: IApplication, workflow_engine: WorkflowEngine):
+
+    def __init__(self, application: IApplication, workflow_name: str):
```

Comment:
We have the active workflow available in the model, so it can already be read from there. These changes aren't necessary. Also, that workflow_engine needs to stay - we're going to need it for actually running the workflows. Might be a bad merge?